CMakeLists.txt

```diff
@@ -34,51 +34,7 @@ if(WIN32 OR MSYS OR MINGW)
   set(CMAKE_INSTALL_BINDIR ".")
   set(PROJECT_NAME "data")
   set(CMAKE_INSTALL_DOCDIR "${CMAKE_INSTALL_DATAROOTDIR}/doc/")
-
   get_filename_component(MINGW_PATH ${CMAKE_CXX_COMPILER} PATH)
```

Comment:
I think this line is no longer needed.

Reply:
Still need the 3 set() and the get_filename_component()

Reply:
Oh you mean for the Qt plugins? Right...

Reply:
yes